docs/Training-PPO.md

```diff
@@ -300,6 +300,16 @@ This option is provided in case you want to initialize different behaviors from
 in most cases, it is sufficient to use the `--initialize-from` CLI parameter to initialize
 all models from the same run.
 
+### (Optional) Advanced: Disable Threading
+
+By default, PPO model updates can happen while the environment is being stepped. To disable this
+behavior, for instance to maintain strict
```

Comment:
The point of this sentence could be a bit clearer. "To disable this, do this." Then, "One may want to do this to maintain the strict on-policyness of PPO..." etc.

Reply:
Updated the docs - hopefully the new one is clearer